exe_2.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+
+def evenly_distributed()->np:

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html